docs/pages/admin-guides/access-controls/rbac/resource-rbac.mdx

```diff
@@ -0,0 +1,108 @@
+
+---
+title: Getting Started With Per-Resource Role-Based Access Control
+description: Implementing fine-grained access control with RBAC.
+---
+
+
+## What is per-resource RBAC?
+
+This granular approach allows you to grant more precise management of user permissions.
+
+Unlike broader, traditional role-based permissions, which can apply to all resources within a namespace or across an entire infrastructure, per-resource RBAC enables administrators to control user access on a per-resource basis. 
+This feature is especially valuable in complex environments with diverse user groups, where some users need restricted access to sensitive systems while others may require broad access.
+
+## Use cases
```

Comment:
The use cases text is pretty extensive. For me, if a user understands the way per-resource RBAC works and wants to set it up, they should understand whether it applies to their own use case. I don't think we need this section.

Reply:
Trimmed! Part of my interpretation of this issue [Provide simple rbac starter example  #39241]() was that additional per-resource examples were requested. I moved 'Best practices' (requested by Xin) to the getting-started page, as it makes more sense there.